### DISCUSSION TITLE
GNSS Station Network processing

### DISCUSSION THREAD

**Discussion by FFFFFG-IGG** [groops-devs/groops#78] [Q&A] [answered]
### Description

Hello developers,
  I'd like to process IGS stations  with fixed satellite products, and get static positions. In this case, should I add  no-net constrains to the normal equation? Since no-net constraints are required when estimating satellite orbit, I'm not sure whether no-net constraint would impact(improve) static position estimation when fix satellite products.
  By the way, station postion should be tightly constrained when estimating satellite products, I haven't found that constraint. Is that no-net constraints?
Many thanks!

### Operating systems

- [ ] Linux
- [X] Windows
- [ ] macOS
- [ ] Other (please specify in the description)

**Comment by zhedumi** [comment, marked as answer]
Hi,

Please correct me if I misunderstand what you want to do but you have satellite clocks, orbits, phase and code biases I presume and just want to compute the daily static position of a set of stations correct?
This is basically just doing [PPP](https://groops-devs.github.io/groops/html/cookbook.gnssPpp.html) within groops. The only thing you have to adapt is changing the parametrization from kinematicPositions to [staticPositions](https://groops-devs.github.io/groops/html/gnssParametrizationType.html#staticPositions) and make it unconstrained within the no-net constraints (=0 value meaning it is unconstrained). 

Regarding the no-net constraints and parametrization information can be found within [Strasser 2022](https://pure.tugraz.at/ws/portalfiles/portal/40917089/Strasser_2022_Doctoral_Thesis_Published_Version.pdf) especially chapter 6.9.1 regarding the no-net constraints.

Best regards,
Patrick
